tests/sweep_framework/sweeps/data_movement/concat/concat_pytorch2.py

```diff
@@ -4194,3 +4195,29 @@ def run(
     output_tensor = ttnn.to_torch(result_tensor)
 
     return [check_with_pcc(torch_output_tensor, output_tensor, 0.999), e2e_perf]
+
+
+@pytest.mark.parametrize("concat_spec", parameters["nightly"]["concat_specs"])
```

Comment:
this was the pytest work around that you were using while sweeps was broken right, you can probably remove it now

Reply:
I think it would be odd to have it on main. The sweeps would open pytest every time they run right?

Reply:
Would it be acceptable to just have it commented out?

Reply:
Will it actually be picked up by runner.py? Commenting it out is def an acceptable solution if so.

Reply:
hm good question. @jdesousa-TT thoughts?